src/cache/cache_las.c

```diff
@@ -18,6 +18,25 @@
 	(WT_SESSION_IGNORE_CACHE_SIZE | WT_SESSION_READ_WONT_NEED |	\
 	WT_SESSION_NO_RECONCILE)
 
+/*
+ * __las_timing_stress --
+ *	Optionally add delay to simulate the race conditions in lookaside
+ * sweep for debug purposes. The purpose is to uncover the race conditions in
+ * lookaside sweep.
+ */
+static void
+__las_timing_stress(WT_SESSION_IMPL *session)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	/* We only want to sleep when lookaside sweep race flag is set. */
+	if (FLD_ISSET(conn->timing_stress_flags,
+	    WT_TIMING_STRESS_LOOKASIDE_SWEEP))
+		__wt_sleep(0, 100 * WT_THOUSAND);
```

Comment:
It would be helpful to use a macro and add a comment:
```
/* discuss why  a tenth of a second or 1/1000th of a second were chosen, etc. */
#define STRESS_SLEEP_LONG (100 * WT_THOUSAND)
#define STRESS_SLEEP_SHORT WT_THOUSAND
```

Reply:
Currently I just considered (100 * WT_THOUSAND) based on other timing stress tests.
Request your suggestion on what conditions we can change the value from STRESS_SLEEP_LONG to STRESS_SLEEP_SHORT.

Reply:
Sorry I was not clear. I think we should just use STRESS_SLEEP_LONG in place of 100 * WT_THOUSAND in the several places we use it for the stress testing, including this new one and the several page split stress places. I do not intend that you change or shorten the value.